sentinel-extension/sentinel-prometheus-metric-exporter/README.md

```diff
@@ -0,0 +1,98 @@
+# Sentinel Prometheus Exporter
+
+Sentinel Prometheus Exporter is a module which provides the Sentinel metrics data for prometheus.
+
+You can integrate it into your Sentinel application, and then get the sentinel metrics in your prometheus.
+
+## How it works
+
+when the prometheus server collect the sentinel metrics,it get metrics from sentinel logs
+![image](https://github.com/alibaba/Sentinel/assets/71377602/2982209b-a3c7-403b-ae50-1dc7a17f90b7)
+
+## How to use
+
+To use Sentinel Prometheus Exporter, you should add the following dependency:
+
+### 1. add sentinel-prometheus-exporter
+
+```xml
+<dependency>
+    <groupId>com.alibaba.csp</groupId>
+    <artifactId>sentinel-prometheus-metric-exporter</artifactId>
+    <version>x.y.z</version>
+</dependency>
+```
+
+### 2. add prometheus dependency
+
+```xml
+<dependency>
+    <groupId>io.prometheus</groupId>
+    <artifactId>simpleclient</artifactId>
+    <version>0.3.0</version>
+</dependency>
+```
+
+```xml
+<dependency>
+    <groupId>io.prometheus</groupId>
+    <artifactId>simpleclient_httpserver</artifactId>
+    <version>0.3.0</version>
+</dependency>
+```
+
+### 3. set prometheus.yml with fetch config
+
+```yaml
+scrape_configs:
+- job_name: 'sentinelMetrics'
+  static_configs:
+    - targets: ['localhost:20001']
```

Comment:
IMO, it would be better to add a tip that the port needs to be the same as the value in the configuration (csp.sentinel.prometheus.fetch.port)

Reply:
./sentinel-extension/sentinel-prometheus-metric-exporter/README.md:52: MD009 Trailing spaces
./sentinel-extension/sentinel-prometheus-metric-exporter/README.md:51: MD031 Fenced code blocks should be surrounded by blank lines

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md